draft-ietf-cose-x509.xml

```diff
@@ -165,14 +171,20 @@ obtained from any of these methods still need to validate it.
         <dt>x5t:</dt>
         <dd>
           <t>
-            This header parameter provides the ability to identify an X.509 certificate by a hash value (a thumbprint).
+            This header parameter provides the ability to identify an end-entity X.509 certificate by a hash value (a thumbprint).
```

Comment:
Shorter more direct text:  "This header parameter identifies the end-entity certificate by a hash value (a thumbprint)."

Reply:
Good I changed according to your suggestion